### DISCUSSION TITLE
Best way to learn git commands

### DISCUSSION THREAD

**Discussion by Ashedamahraomai** [community/community#113334] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

What is the best way to learn Git commands and their workings?

**Comment by deka27** [comment]
I prefer [Oh my Git](https://ohmygit.org/) to learn in a interactive way.

There are some other suggestions as well. [Link](https://www.reddit.com/r/git/comments/13ahijy/any_recommendations_on_best_resources_to_learn/)

**Comment by ettaboyle** [comment]
We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. 

Any future violations may result in an indefinite block from the Community. Thanks for understanding.

**Comment by ettaboyle** [comment]
@deka27 & @Ashedamahraomai I've logged a temporary ban on both of your accounts as a warning. You can still view discussions in this community, you just won't be able to post new content for 7 days. 

In our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md), we ask that members contribute in a positive and constructive way. This type of post is considered "off topic" for our Community Discussions. Any repeat and future violations may result in an indefinite block from the Community. 

Thanks for understanding.
